src/main/java/MarkCommand.java

```diff
@@ -0,0 +1,29 @@
+package duke;
+
+/**
```

Comment:
Would it be better if the MarkCommand class could mark or unmark tasks? Thereby eliminating the need for an UnmarkTask class.